### PR TITLE
Change catching type in Repository constructor from \Exception to \Thowable

### PR DESCRIPTION
**Pull request by veeeeeRySeXy**
Change catching type in Repository constructor from \Exception to \Thowable

Add description to Repository constructor

### CODE REVIEW

**Comment by populov** [comment]
If you touch it - test it. Create unit test for repository creation, ex. 2 cases:
1. when no model set 
2. model class is incorrect

**Comment by veeeeeRySeXy** [comment]
Got it.

**Comment by veeeeeRySeXy** [comment]
@populov Merge request was updated. Watch it, please.

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+------
+Add Mocker helper yo use db connections mocks instead sqlite:memory.
+Add mockery/mockery in composer dependencies.
+Add base repository test.
+
+2.1.2
+------
+Update accepted exception type in RepositoryException from \Exception to \Throwable.
+
+2.1.1
```
Comment (populov) [inline CHANGES.md]: Do we need to bump 3 build versions in single release? I think no. You can describe all 3 changes under 1 version.

**Review by populov — APPROVED** [review]

**Pull request merged**